src/DssSpell.t.sol

```diff
@@ -915,7 +965,149 @@ contract DssSpellTest is DSTest, DSMath {
         checkSystemValues(afterSpell);
 
         checkCollateralValues(afterSpell);
+    }
+
+	function testSpellIsCast_UNIV2WBTCETH_INTEGRATION() public {
+        vote();
+        scheduleWaitAndCast();
+        assertTrue(spell.done());
+
+        bytes32 ilk = "UNIV2WBTCETH-A";
+
+        lpPipWbtc.poke();
+        hevm.warp(now + 3601);
+        lpPipWbtc.poke();
+        spot.poke(ilk);
+
+        // Check median matches pip.src()
+        assertEq(lpPipWbtc.src(), address(lpWbtc));
+        assertEq(lpPipWbtc.orb0(), address(0xe0F30cb149fAADC7247E953746Be9BbBB6B5751f));
```

Comment:
might be useful to use `addr.addr("WBTC")` in these tests so they're easier to audit

Reply:
actually this is the medianizer nvm